alibuild_helpers/cmd.py

```diff
@@ -91,11 +91,12 @@ def __enter__(self):
       cmd += [self._docker_image, "sleep", "inf"]
       self._container = getoutput(cmd).strip()
 
-    def getstatusoutput_docker(cmd):
+    def getstatusoutput_docker(cmd, cwd=None):
       if self._container is None:
-        return getstatusoutput("{} -c {}".format(BASH, quote(cmd)))
+        return getstatusoutput("{} -c {}".format(BASH, quote(cmd)), cwd=cwd)
       return getstatusoutput("docker container exec {} bash -c {}"
-                             .format(quote(self._container), quote(cmd)))
+                             .format(quote(self._container), quote(cmd)),
```

Comment:
Does this actually work? It will change the cwd of where the container executes stuff, not the one of the actual command, no?

Reply:
It doesn't make much sense to change it for docker in this case, but in any case the docker container will be disposed of as soon as the checks are done. 

This change mostly matters to not leave random stuff on the host

Reply:
Is this addressed?

Reply:
Can you try?